src/registrar/management/commands/delete_expired_domains_not_setup.py

```diff
@@ -0,0 +1,99 @@
+from django.core.management import BaseCommand
+from registrar.models import Domain, UserDomainRole
+import logging
+import argparse
+from django.utils import timezone
+from datetime import timedelta
+from registrar.utility.email import EmailSendingError, send_templated_email
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """If a domain is in the "Unknown" or "DNS Needed" state, and has been expired for 7 days,
```

Comment:
```suggestion
    """Domains that, (1) have DNS status "Unknown" or "DNS Needed" and (2) are 7+ days past their expiration date, are marked "DELETED" in the registrar and deleted in the registry.
```
Thought this would specify we're given the domain "deleted" status in the registrar but are actually deleting it from the registry